<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormVisionWindow</class>
 <widget class="QWidget" name="FormVisionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>344</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FormVisionWindow</string>
  </property>
  <widget class="QLabel" name="lb_x">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_z">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_x_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_x_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_x_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_x_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PIT:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_roll">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_x_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ROL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_x_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>YAW:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pitch">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_yaw">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
